README.md

```diff
@@ -25,40 +23,14 @@ The app requires a Git token for access to private repos (currently limited to `
 
 This will then be mounted as a secret in docker, and used to authenticate against the package repo.
 
-## Available Scripts
-
-In the project directory, you can run:
-
-### `yarn start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3010](http://localhost:3010) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```

Comment:
❓ this still runs the web component tests I believe

Reply:
re-added